es55x/src/main/scala/tech/beshu/ror/es/request/context/BaseEsRequestContext.scala

```diff
@@ -145,6 +145,9 @@ abstract class BaseEsRequestContext[B <: BlockContext](esContext: EsContext,
     }
   }
 
+  override def isRorAdminRequest: Boolean = {
```

Comment:
1. let's remove it from here 
2. let's add `isRorAction` method to Action class
3. at the moment we have 4 ROR actions
4. let's use it like this in proper rules: `blockContext.requestContext.action.isRorAction`